src/tracer.js

```diff
@@ -95,10 +100,30 @@ export default class Tracer {
 
         span.addTags(tags);
         span.addTags(internalTags);
+
+        // emit metrics
+        this._metrics.spansStarted.increment(1);
+        if (span.context().isSampled()) {
+            this._metrics.spansSampled.increment(1);
+            if (!hadParent) {
+                this._metrics.tracesStartedSampled.increment(1);
+            } else if (rpcServer) {
+                this._metrics.tracesJoinedSampled.increment(1);
+            }
+        } else {
+            this._metrics.spansNotSampled.increment(1);
+            if (!hadParent) {
```

Comment:
the `parent` in L179 is not exactly the right condition, because with jaeger-debug-id you would still have a parent context, but it's not a real parent. You may notice how in Go I have a separate variable which is only set to true if the parent context is "valid"